sync/class.jetpack-sync-module-full-sync.php

```diff
@@ -267,6 +279,33 @@ public function get_action_name( $queue_item ) {
 		return false;
 	}
 
+	public function get_action_totals( $queue_item ) {
```

Comment:
Could you add a quick docblock for this function and the other new one added? (I'm trying to slowly get everything to coding standards as things are touched in existing files)

Reply:
Great call, thanks! Added docblocks for the new methods here: 8641517ec3ea434912663488682aef82f79310a8